slack/channels.md

```diff
@@ -27,6 +27,13 @@ Please note that like any open source project, the main language on this Slack i
 - Status: Public
 - Language: EN
 
+**#community-help**
+- Description: A place for folks who need helps while using the PrestaShop project to build online shops. Please open an issue on the devdocs when this is about a missing information.
```

Comment:
```suggestion
- Description: A place for folks who need help while using the PrestaShop project to build online shops. Please open an issue on the devdocs when this is about a missing information.
```

You seem to like the word "folks" 🤔are you sure you want to keep it instead of "peopl"

If you're sure, OK 👍I just ask for confirmation

Reply:
"Folks" is used more and more in inclusive talking and writing in English, in open source communities but not only. That's why I try to use more and more too.
If you feel people is ok too (not too much like "les gens") it's ok to use people.